src/views/notifications/components/addedModeratorNotification.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import * as React from 'react';
+import { parseEvent, parseNotificationDate, parseContext } from '../utils';
+import { ActorsRow } from './actorsRow';
+import { NotificationCard, TextContent, JoinContext, Content } from '../style';
+import Icon from '../../../components/icons';
+import {
+  CardLink,
+  CardContent,
+} from '../../../components/threadFeedCard/style';
+import markSingleNotificationSeenMutation from 'shared/graphql/mutations/notification/markSingleNotificationSeen';
+
+type Props = {
+  notification: Object,
+  currentUser: Object,
+  markSingleNotificationSeen?: Function,
+  markSingleNotificationAsSeenInState?: Function,
+};
+
+export class AddedModerator extends React.Component<Props> {
+  render() {
+    const { notification, currentUser } = this.props;
+
+    const actors = parseActors(notification.actors, currentUser, true);
+    const event = parseEvent(notification.event);
+    const date = parseNotificationDate(notification.modifiedAt);
+    const context = parseContext(notification.context);
+
+    return (
+      <NotificationCard>
+        <CardLink to={`/${notification.context.payload.slug}`} />
+        <CardContent>
+          <JoinContext>
+            <Icon glyph="member-add" />
```

Comment:
Could someone provide me with an icon for this notification? :)

Reply:
Feel free to re-use this member add icon for this one :)